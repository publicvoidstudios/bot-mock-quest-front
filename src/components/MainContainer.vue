<template>
  <div class="container grid grid-cols-1 sm:grid-cols-1 mx-auto h-[100dvh]">
    <slot></slot>
  </div>
</template>