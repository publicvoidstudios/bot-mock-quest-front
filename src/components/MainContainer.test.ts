import { mount } from '@vue/test-utils';
import {describe, it, expect} from "vitest";
import MainContainer from './MainContainer.vue';

describe('MainContainer.vue', () => {
    const wrapper = mount(MainContainer);

    it('renders correctly', () => {
        expect(wrapper.element).toMatchSnapshot();
    });
});