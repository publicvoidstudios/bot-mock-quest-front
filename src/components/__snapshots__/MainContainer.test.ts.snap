// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MainContainer.vue > renders correctly 1`] = `
<main>
  
  
</main>
`;
